### PR TITLE
Added the checkbox name to a hidden unchecked value element

### PR DESCRIPTION
**Pull request by jelofson**
When setting the hidden element that holds the unchecked value, it must have the same name as the checkbox. If not, it has no purpose.

### CODE REVIEW

**Comment by jelofson** [comment]
Just a few updates to the docs. I also updated the checkbox helper to move the hidden unchecked element outside of the label. It is now valid html. Updated tests

File: README.md [hunk 1]
```diff
@@ -93,51 +93,74 @@ checkbox
 ```php
 <?php
 $helper->input(array(
-    'type' => "checkbox",
-    'name' => "name",
-    'value' => "value",
+    'type' => 'checkbox',
+    'name' => 'foo',
+    'value' => 'bar',
     'attribs' => array(
```
Comment (harikt) [inline README.md:99]: I have a feeling you should not have changed the double quote to single quote.

**Comment by harikt** [comment]
Thank you.

@pmjones will check and merge.

File: README.md (hunk 1 above)
Comment (jelofson) [inline README.md:99]: Any particular reason? There seems to be a real mix of double and single and I was going for consistency. Although I certainly didn't change all of them :)

Comment (harikt) [inline README.md:99]: Why I thought was when we are sending a PR with a failure to test, the style should not get into it. That makes the life to look into the test easier. Myself included ;) .

**Comment by harikt** [comment]
@jelofson so I did tested the validation you have mentioned. 

You are right on this. 

Thank you and sorry for the delay. Merging this.

**Comment by jelofson** [comment]
No problem

**Pull request merged**